course02/homework00/danielellis/vector.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <array>
+#include <exception>
+
+struct RanOutOfMemoryException : public std::exception
+{
+};
+
+class VectorInt
+{
+public:
+	size_t size() const
```

Comment:
This will compile on most platforms, but strictly speaking `size_t` is a C type that you will find in stddef.h and might not be defined in cstddef. It's better to stick to `std::size_t`.